### DISCUSSION TITLE
slugifier generates slugs with 3 dashes in a row when a dash exists in the product name. should this change?

### DISCUSSION THREAD

**Discussion by jezzzm** [vendurehq/vendure#3169] [General]
quite a few of our products happen to include a dash `-` in the product name, so when our staff create products, the slug generated often has too many dashes.

correction is frequently missed as the dashes are often out of view of the slug input, so you'd have to scroll sideways within it to verify the generated slug.

**is it safe to assume that this output would never be desired in any locale?**
e.g.
name: `The Kutopia Collection - Complete Bundle`
slug: `the-kutopia-collection---complete-bundle`

it'd be a pretty quick update to the normalizer with `.replace(/-{1,}/g, '-')` (or dynamically - the dash replaced with whatever separator is applied)

**Comment by michaelbromley** [comment]
Update: fix merged, thanks!
